chain-impl-mockchain/src/vote/manager.rs

```diff
@@ -237,31 +237,17 @@ impl ProposalManager {
     where
         F: FnMut(&VoteAction),
     {
-        use std::convert::TryInto;
         let tally = self.tally.as_ref().ok_or(TallyError::NoEncryptedTally)?;
         let (encrypted_tally, total_stake) = tally.private_encrypted()?;
         let state = encrypted_tally.state();
         let table_size = total_stake.0 as usize / self.options.choice_range().len();
-        let private_result = chain_vote::result(total_stake.0, table_size, &state, shares);
+        let private_result = chain_vote::tally_result(total_stake.0, table_size, &state, shares)
```

Comment:
The `verify` function will be used here when the necessary info will be in the transaction, right?

Reply:
Yes, not changing that in this PR.